pandas-stubs/io/sql.pyi

```diff
@@ -28,6 +28,10 @@ _SQLConnection = Union[
     sqlite3.Connection,
 ]
 
+_SQLStatement = Union[
+    str, sqlalchemy.sql.expression.Selectable, sqlalchemy.sql.elements.TextClause
```

Comment:
need to `import sqlalchemy.sql.elements`

Reply:
Thanks.  I agree expression is the right endpoint, also based on https://docs.sqlalchemy.org/en/14/core/sqlelement.html#sqlalchemy.sql.expression.TextClause